<launch>
    <include file = "$(find r6g_steppermotor)/launch/variables.launch">
        <arg name = "csvpath" value = "$(arg csvpath)"/>
        <arg name = "port" default = "$(arg port)" />
        <arg name = "delay" default = "$(arg delay)" />
        <arg name = "delaySerial" default = "$(arg delaySerial)" />
    </include>  

    <node name = "Trayectoria" pkg = "r6g_steppermotor" type = "Trayectoria.py" args = "$(arg csvpath)" output = "screen" />
    <node name = "ReadCSV" pkg = "r6g_steppermotor" type = "ReadCSV.py" args = "$(arg csvpath)" output = "screen" launch-prefix = "bash -c 'sleep $(arg delay); $0 $@' "/>
    <node name = "SerialSending" pkg = "r6g_steppermotor" type = "SerialSending.py" args = "$(arg port)" output = "screen" launch-prefix = "bash -c 'sleep $(arg delaySerial); $0 $@'" />
</launch>